CodeStarterCamp_Week2/week2_step2.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  week2_step2.swift
+//  CodeStarterCamp_Week2
+//
+//  Created by jjudy on 2023/09/04.
+//
+
+import Foundation
+
+func createLottoNumbers() -> Set<Int> {
+    var lottoSet: Set<Int> = Set()
+    
+    while lottoSet.count < 6 {
+        lottoSet.insert(Int.random(in: 1...45))
+    }
+    
+    return lottoSet
+}
+
+func checkSameLottoNumbers(myLottoNumbers: [Int], setLottoNumbers: Set<Int>) {
+    var sameNumArray: Array<Int> = [Int]()
+    
+    for num in myLottoNumbers {
+        if (setLottoNumbers.contains(num)) {
+            sameNumArray.append(num)
+        }
+    }
+    
+    // sameNumArray.count > 0 과 같은 의미로 isEmpty를 부정문으로 사용해 조건 설정
+    if(!sameNumArray.isEmpty) {
+        print("축하합니다! 겹치는 번호는 ", terminator: "")
+        print(sameNumArray.map{String($0)}.joined(separator:", "), terminator: "")
+        print(" 입니다!")
+        
+        // 위 표현 한 줄로 작성해보기
+        print("축하합니다! 겹치는 번호는", sameNumArray.map{String($0)}.joined(separator: ", "), "입니다!")
```

Comment:
```suggestion
    if(!sameNumArray.isEmpty) {
        let result = sameNumArray.map{String($0)}.joined(separator: ", ")        
        print("축하합니다! 겹치는 번호는 \(result) 입니다!")
```
뭐 이렇게 가도 좋을 것 같습니다.
print문을 한줄로 줄이면 좋은 이유는 나중에 UI를 그릴 때 Text에 색깔, 폰트 등을 한번에 설정할 수 있는 장점이 있어서 연습 겸 제안을 해본 것 입니다. 잘하셨어요👍